src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -8,10 +8,11 @@
 from sentry.utils import email
 
 
-def is_issue_link_integration(sentry_app):
-    """Determine if the sentry app supports issue linking"""
+def has_ui_component(sentry_app):
+    """Determine if the sentry app supports issue linking or stack trace linking."""
+    ui_components = ["issue-link", "stacktrace-link"]
```

Comment:
`src/sentry/api/endpoints/sentry_app_interaction.py` has the same logic:
```py
COMPONENT_TYPES = ["stacktrace-link", "issue-link"]
...
if component_type is None or component_type not in COMPONENT_TYPES:
```
Is there somewhere where we can put this function that is accessible to both endpoints?